src/components/safe-apps/SafeAppLandingPage/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect } from 'react'
+import { Paper } from '@mui/material'
+import Grid from '@mui/material/Unstable_Grid2'
+import { OVERVIEW_EVENTS, SAFE_APPS_EVENTS, trackEvent } from '@/services/analytics'
+import { useSafeAppFromBackend } from '@/hooks/safe-apps/useSafeAppFromBackend'
+import { useSafeAppFromManifest } from '@/hooks/safe-apps/useSafeAppFromManifest'
+import { SafeAppDetails } from '@/components/safe-apps/SafeAppLandingPage/SafeAppDetails'
+import { TryDemo } from '@/components/safe-apps/SafeAppLandingPage/TryDemo'
+import { UseApp } from '@/components/safe-apps/SafeAppLandingPage/UseApp'
+import useWallet from '@/hooks/wallets/useWallet'
+import { AppRoutes } from '@/config/routes'
+import { SAFE_APPS_DEMO_SAFE_MAINNET } from '@/config/constants'
+import useOnboard from '@/hooks/wallets/useOnboard'
+import { Errors, logError } from '@/services/exceptions'
+import useOwnedSafes from '@/hooks/useOwnedSafes'
+import { useAppSelector } from '@/store'
+import { selectChainById } from '@/store/chainsSlice'
+
+type Props = {
+  appUrl: string
+  chainId: string
+}
+
+const SafeAppLanding = ({ appUrl, chainId }: Props) => {
+  const { safeApp, isLoading } = useSafeAppFromManifest(appUrl, chainId)
+  const [backendApp, , backendAppLoading] = useSafeAppFromBackend(appUrl)
+  const wallet = useWallet()
+  const onboard = useOnboard()
+  const ownedSafes = useOwnedSafes()[chainId] ?? []
+  const chain = useAppSelector((state) => selectChainById(state, chainId))
+  // show demo if the app was shared for mainnet or we can find the mainnet chain id on the backend
+  const showDemo = chainId === '1' || !!backendApp?.chainIds.includes('1')
```

Comment:
I think we should move the `1` out into a constant as this is easily overlooked.